app/models/thredded/topic.rb

```diff
@@ -15,6 +15,23 @@ class Topic < ActiveRecord::Base
 
     scope :order_sticky_first, -> { order(sticky: :desc) }
 
+    scope :followed_by, lambda { |user|
+      joins('INNER JOIN thredded_user_topic_follows ON thredded_user_topic_follows.topic_id = thredded_topics.id')
+        .where('thredded_user_topic_follows.user_id = ?', user.id)
+    }
+    scope :unread_by, lambda{ |user|
```

Comment:
Hadn't seen this comment before I went ahead and merged it using the Github merge&squash thing.